File: C/Array/prg8.c
#include<stdio.h>

void main() {

        int a[3] = {1, 2, 3};
        int b[3] = {1, 2, 3};

        if(a == b)
                printf("Same\n");

        /*
        a = &a[0] and b = &b[0] then
        Obviously different because &a[0] != &b[0]
        */

        else
                printf("Different\n");
}

File: Java/Polymorphism/Overloading/prg8.java
class Custom1 {

        void fun(Thread t, int a) {

        }

        int fun(int a, Thread t) { //sequence is different, works

                return 1;
        }

        char gun(Thread t, int a) {

                return 'A';
        }

        //Tip-
        //return type does not matter in overloading a method
        //contrary to overriding, where return type must be same or co-variant
        String fun(Thread t, int a, char c) {

                return "Abhijeet";
        }

        /*void fun(Thread t, int a) {

        }*/
}

class Custom2 {

        void fun(Thread t, int a) {

        }
}
